low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -185,20 +194,32 @@ def install_files(self, PRODUCT):
         # Onward LR2, consul will be abstracted out and it won't exit as hard dependeny of SSPL
         if PRODUCT == "LDR_R1":
             if not os.path.exists(REPLACEMENT_NODE_ENV_VAR_FILE):
-                config_from_salt = "%s/sspl_fetch_config_from_salt.py %s %s" % (self._script_dir,
-                                                                                self.ENVIRONMENT,
-                                                                                PRODUCT)
-                output, error, returncode = SimpleProcess(config_from_salt).run()
+                config_from_salt = "%s/sspl_fetch_config_from_salt.py %s %s" % (
+                    self._script_dir,
+                    self.ENVIRONMENT,
+                    PRODUCT)
+                output, error, returncode = SimpleProcess(
+                    config_from_salt).run()
                 if returncode != 0:
                     raise SetupError(returncode, error, config_from_salt)
 
         # Skip this step if sspl is being configured for node replacement scenario as rabbitmq cluster is
         # already configured on the healthy node
         # Configure rabbitmq
         if not os.path.exists(REPLACEMENT_NODE_ENV_VAR_FILE):
-            setup_rmq = Conf.get(
-                SSPL_CONFIG_INDEX, "RABBITMQCLUSTER>setup_rmq")
-            if setup_rmq:
-                dbus_service.start('rabbitmq-server.service')
-                sspl_rabbitmq_reinit.main(PRODUCT)
-
+            message_types = set([Conf.get(SSPL_CONFIG_INDEX,
+                                          "RABBITMQINGRESSPROCESSOR"
+                                          ">message_type"),
+                                 Conf.get(SSPL_CONFIG_INDEX,
+                                          "RABBITMQEGRESSPROCESSOR"
+                                          ">message_type")])
+            mb = MessageBus()
+            mbadmin = MessageBusAdmin(mb, admin_id='admin')
+            try:
+                mbadmin.register_message_type(message_types=message_types,
+                                              partitions=1)
+            except MessageBusError as e:
+                if "ALREADY_EXISTS" in e.desc:
+                    pass
```

Comment:
log that topic exists

Reply:
Where are we going to store logs of these files(provisioner interface files). If we are not storing, logging here will not log anything. Also print statements should not be there in library code as per Ujjawal.

Reply:
oh right, this is provisioning stage. script would print on screen itself.